### DISCUSSION TITLE
How Do I Enable the Subdag to Execute Asynchronously?

### DISCUSSION THREAD

**Discussion by lixiaoyong12** [apache/airflow#15227] [Q&A] [answered]
Two DAGs are executed asynchronously. After the parent DAG is executed, the next cycle is executed, and the child DAG executes the previous cycle.

**Comment by turbaszek** [comment]
@lixiaoyong12 I'm not sure I understand the problem. Can you describe or draw your DAG?

**Reply by lixiaoyong12** [reply]
I have two DAGs, DAG A and DAG B; if DAG A finishes, DAG B is triggered, DAG A continues with the next batch of tasks. For example, DAG A executes the 2:00 task, DAG B executes the 1:00 task.DAG B depends on DAG A's result

**Comment by lixiaoyong12** [comment, marked as answer]
I have two DAGs, DAG A and DAG B; if DAG A finishes, DAG B is triggered, DAG A continues with the next batch of tasks.  For example, DAG A executes the 2:00 task, DAG B executes the 1:00 task.

**Reply by lixiaoyong12** [reply]
DAG B depends on DAG A's result
